Data/Primitive/Array.hs

```diff
@@ -30,7 +30,10 @@ module Data.Primitive.Array (
 import Control.Monad.Primitive
 
 import GHC.Base  ( Int(..) )
-import GHC.Prim
+import GHC.Exts
+#if (MIN_VERSION_base(4,7,0))
+  hiding (toList)
+#endif
 import qualified GHC.Exts as Exts
 #if (MIN_VERSION_base(4,7,0))
 import GHC.Exts (fromListN, fromList)
```

Comment:
There's a couple of spots in the code where we already import `GHC.Exts` in the same module, such as in this line. It would be good to consolidate these imports with the newly-created `GHC.Exts` imports that this patch introduces.